superset/viz.py

```diff
@@ -865,6 +865,12 @@ def query_obj(self) -> QueryObjectDict:
             raise QueryObjectValidationError(_("Please choose at least one metric"))
         if set(groupby) & set(columns):
             raise QueryObjectValidationError(_("Group By' and 'Columns' can't overlap"))
+        sort_by = self.form_data.get("timeseries_limit_metric")
```

Comment:
I am curious, why use this control in a  not time-series chart.

Reply:
I think this is just from legacy form_data field names. We'd need a db migration if we want to rename it.

Superset has a timeseries-first UI. Each pivot group can be considered a series since the base query has the ability to aggregate by time + pivot column.

Reply:
Wait a minute, since this is a new field for pivot table and https://github.com/apache-superset/superset-ui/pull/952 hasn't been merged, maybe we CAN rename it to something else?

Reply:
Or we can keep it as `timeseries_limit_metric` to be able to reuse the existing code and do a wholesale migration later.

Reply:
cc @villebro what do you think?

Reply:
I was super confused by the name yesterday, and agree we need to get rid of it. I'm ok both ways, but I think it might be simpler to do a bulk rename later to avoid partially migrated names.